src/components/chip-set/chip.types.ts

```diff
@@ -49,4 +49,10 @@ export interface Chip<T = any> {
      * The value of the badge. Only valid for `filter`.
      */
     badge?: number;
+
+    /**
+     * If supplied, the chip will render an `<a>` element, using the supplied
```

Comment:
Isn't it always rendering an `<a>` element in this implementation?

Reply:
Yes, it is actually… I wrote this before I implemented it, and it turned out that rendering an `<a>` without a `href` was much easier that switching between rendering an `<a>` or a `<div>`, and it had the same result…

I'll update the description 👍

Reply:
⚡